frontend/views/map.js

```diff
@@ -88,7 +89,19 @@ module.exports = Backbone.View.extend({
         }
         $('.no-search-results').hide();
         $('#map_container').show();
-        $('.results-truncated').toggle(search.has_more);
+
+        if (services.length < search.total_results) {
+            msg = i18n.t("Service-Map.More-Results", {
+                // ARGH!  "count" is a magic var name to i18n and won't work here,
```

Comment:
Just so I'm understanding this comment correctly, this is meant to explain why the variable is named `thecount` rather than simply `count` right?